modin/pandas/test/test_api.py

```diff
@@ -249,6 +250,35 @@ def test_series_str_api_equality():
     )
 
 
+def test_series_cat_api_equality():
+    modin_dir = [obj for obj in dir(pd.Series().cat) if obj[0] != "_"]
+    pandas_dir = [obj for obj in dir(pandas.Series.cat) if obj[0] != "_"]
+
+    missing_from_modin = set(pandas_dir) - set(modin_dir)
+    assert not len(missing_from_modin), "Differences found in API: {}".format(
+        len(missing_from_modin)
+    )
+    extra_in_modin = set(modin_dir) - set(pandas_dir)
+    assert not len(extra_in_modin), "Differences found in API: {}".format(
+        extra_in_modin
+    )
+
+
+@pytest.mark.parametrize("klass", ["DataFrame", "Series"])
```

Comment:
```suggestion
@pytest.mark.parametrize("class", ["DataFrame", "Series"])
```

Reply:
This is done on purpose as `class` is a reserved word.